components/footer/index.css

```
@@ -1,5 +1,20 @@
 .footer {
-    background-color: var(--chocolate);
-    height: 80px;
+    height: 144px;
     min-height: 80px;
+    color: var( --gray );
+    margin: auto;
```

```suggestion
    display: flex;
    padding: 64px 0;
```

Remove height and margins here, please